src/assets/js/protocol/protocol-menu.js

```diff
@@ -0,0 +1,311 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+(function() {
+    'use strict';
+
+    var Menu = {};
+    var _hoverTimeout;
+    var _hoverTimeoutDelay = 150;
+    var _mqWideNav;
+    var _wideBreakpoint = '768px';
+
+    var _options = {
+        onMenuOpen: null,
+        onMenuClose: null,
+        onMenuButtonClose: null
+    };
+
+    /**
+     * Opens a menu panel.
+     * @param {Object} el - DOM element (`.mzp-c-menu-category.mzp-js-expandable`)
+     * @param {Boolean} animate - show animation when menu panel opens.
+     */
+    Menu.open = function(el, animate) {
+        if (animate) {
+            el.classList.add('animate');
+        }
+
+        el.classList.add('selected');
+        el.setAttribute('aria-selected', true);
+        el.setAttribute('aria-expanded', true);
+
+        if (typeof _options.onMenuOpen === 'function') {
+            _options.onMenuOpen(el);
+        }
+    };
+
+    /**
+     * Closes all currently open menu panels.
+     * Note: on small screens more than one menu can be open at the same time.
+     */
+    Menu.close = function() {
+        var current = document.querySelectorAll('.mzp-c-menu-category.selected');
+
+        for (var i = 0; i < current.length; i++) {
+            current[i].classList.remove('animate');
+            current[i].classList.remove('selected');
+            current[i].setAttribute('aria-selected', false);
+            current[i].setAttribute('aria-expanded', false);
+        }
+
+        if (typeof _options.onMenuClose === 'function' && current.length > 0) {
+            _options.onMenuClose();
+        }
+
+        return current.length > 0;
+    };
+
+    /**
+     * Menu panel close button `click` event handler.
+     * @param {Object} e - Event object.
+     */
+    Menu.onCloseButtonClick = function(e) {
+        e.preventDefault();
+
+        if (typeof _options.onMenuButtonClose === 'function') {
+            _options.onMenuButtonClose();
+        }
+
+        Menu.close();
+    };
+
+    /**
+     * Toggles the open/closed state of a menu panel.
+     * @param {Object} el - DOM element (`.mzp-c-menu-category.mzp-js-expandable`)
+     */
+    Menu.toggle = function(el) {
+        var state = el.classList.contains('selected') ? true : false;
```

Comment:
`.is-selected`?  I think this is the same element that Craig commented on in his review.